client/version.go

```diff
@@ -0,0 +1,102 @@
+// pmm-agent
```

Comment:
this file should be separate structure.

Reply:
could you please add more details.

Reply:
could you please add more details.

Reply:
I mean that `localMySQLVersion`, `xtrabackupVersion` methods doesn't look like it should be part of `Client` structure, my suggestion to make it separate function like we did for `CheckConnection`

Reply:
I see, will do.